etcdctl/ctlv3/command/util.go

```diff
@@ -48,8 +48,23 @@ func addHexPrefix(s string) string {
 }
 
 func argify(s string) []string {
-	r := regexp.MustCompile("'.+'|\".+\"|\\S+")
-	return r.FindAllString(s, -1)
+	r := regexp.MustCompile(`"(\\.|[^"])*"|'[^']*'|[^'"\s]\S*[^'"\s]?`)
```

Comment:
the single quote and non-quote one looks fine to me.

It seems like people use `/"(?:[^"\\]|\\.)*"/` for double quote to deal with backslash.

See http://stackoverflow.com/questions/249791/regex-for-quoted-string-with-escaping-quotes.

Not sure if it necessary for us though, since we will do scanf %q later.